index.html

```diff
@@ -500,22 +500,21 @@ <h2>Licensing</h2>
       </section>
 
 
-<!--
+
       <section id="about">
         <h2>
           About this Charter
         </h2>
         <p>
-          This charter has been created according to <a href="https://www.w3.org/Consortium/Process/#WG-and-IG">section 5.2</a> of the <a href="https://www.w3.org/Consortium/Process/">Process Document</a>. In the event of a conflict between this document or the provisions of any charter and the W3C Process, the W3C Process shall take precedence.
+          This charter has been created according to <a href="https://www.w3.org/Consortium/Process/#GAGeneral">section 3.4</a> of the <a href="https://www.w3.org/Consortium/Process/">Process Document</a>. In the event of a conflict between this document or the provisions of any charter and the W3C Process, the W3C Process shall take precedence.
         </p>
 
         <section id="history">
           <h3>
             Charter History
           </h3>
-          <p class="issue"><b>Note:</b>Display this table and update it when appropriate. Requirements for charter extension history are documented in the <a href="https://www.w3.org/Guide/process/charter.html#extension">Charter Guidebook (section 4)</a>.</p>
 
-          <p>The following table lists details of all changes from the initial charter, per the <a href="https://www.w3.org/Consortium/Process/#CharterReview">W3C Process Document (section 5.2.3)</a>:</p>
+          <p>The following table lists details of all changes from the initial charter, per the <a href="https://www.w3.org/Consortium/Process/#CharterReview">W3C Process Document (section 4.3, Advisory Committee Review of a Charter)</a>:</p>
```

Comment:
I prefer to see a history table that goes back to the beginning of the WG - something that shows the initial charter, the recharter into maintenance mode, and anything in between. The line for this recharter would name the new deliverables and summarize, very briefly, the change in scope.

Reply:
(copying from https://github.com/w3c/vc-wg-charter/issues/55#issuecomment-1024239530)

Do you mean copy the whole of https://www.w3.org/2020/12/verifiable-credentials-wg-charter.html#about into this charter (with an additional line that is there)? I am fine with this, but I am a bit afraid it is a little bit over the top...

Reply:
@samuelweiler I have added the previous change table but, to reduce the size, I have removed the purely administrative changes from the previous table (change of staff contact, change of chairs). I think that should be o.k.

Reply:
You might ask comm for guidance/preferences.  I'd say that if you remove anything, put a note saying "more history can be found in the previous charter", and link it there (also).

Reply:
Frankly, I do not want to loose my time on this. I have re-added those administrative history lines. The table is now complete.